tests/attr/neuron/test_neuron_conductance.py

```diff
@@ -93,6 +100,15 @@ def test_simple_conductance_multi_input_batch_relu(self) -> None:
             (inp3, 5),
         )
 
+    def test_layer_tuple_selector_fn(self) -> None:
+        net = BasicModel_MultiLayer(multi_input_module=True)
+        inp = torch.tensor([[0.0, 6.0, 0.0]])
+        self._conductance_input_test_assert(
+            net, net.multi_relu, inp, lambda x: x[0][:, 1], [0.0, 6.0, 0.0]
+        )
+        lc = LayerConductance(net, net.multi_relu)
+        print(lc.attribute(inp, target=0))
```

Comment:
do we need this printout ? Did you wanted to assert with the results from Layer Conductance ?

Reply:
Whoops, good catch thanks! Was initially going to do that, but then made it a separate test, forgot to remove this.